src/dex/algebra/algebra-e2e.test.ts

```diff
@@ -181,4 +181,28 @@ describe('Algebra', () => {
       );
     });
   });
+
+  describe('CamelotV3', () => {
+    const dexKey = 'ZyberSwapV3';
```

Comment:
Why 'ZyberSwapV3'? I can not find USDC/DAI for CamelotV3

Reply:
obviously wrong , solved